src/NClient/NClient.Abstractions/Providers/Transport/Responses/Response.cs

```diff
@@ -34,6 +34,12 @@ public new Response<TData> EnsureSuccess()
             base.EnsureSuccess();
             return this;
         }
+
+        public void Deconstruct(out TData? data, out IContent? content)
```

Comment:
The date is a deserialized content. Why would you need both at once?

Reply:
Delete.